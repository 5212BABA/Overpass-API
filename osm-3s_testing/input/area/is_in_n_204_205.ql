node(20495);is_in;out geom;
