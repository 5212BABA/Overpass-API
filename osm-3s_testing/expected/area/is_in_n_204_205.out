<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <way id="204">
    <bounds minlat="51.2500001" minlon="179.9990001" maxlat="51.2510001" maxlon="-179.9990001"/>
    <nd ref="20401" lat="51.2500001" lon="179.9990001"/>
    <nd ref="20402" lat="51.2500001" lon="-179.9990001"/>
    <nd ref="20403" lat="51.2510001" lon="-179.9990001"/>
    <nd ref="20404" lat="51.2510001" lon="179.9990001"/>
    <nd ref="20401" lat="51.2500001" lon="179.9990001"/>
  </way>
  <way id="205">
    <bounds minlat="51.2500001" minlon="179.9990001" maxlat="51.2700001" maxlon="-179.9990001"/>
    <nd ref="20501" lat="51.2500001" lon="179.9990001"/>
    <nd ref="20502" lat="51.2500001" lon="-179.9990001"/>
    <nd ref="20503" lat="51.2700001" lon="-179.9990001"/>
    <nd ref="20504" lat="51.2700001" lon="179.9990001"/>
    <nd ref="20501" lat="51.2500001" lon="179.9990001"/>
  </way>

</osm>
